From ae4622baab4923bdbc30273bb35c7479c92e2e77 Mon Sep 17 00:00:00 2001
From: "Gopal, Puunithaaraj" <puunithaaraj.gopal@intel.com>
Date: Wed, 30 Jan 2019 23:44:12 +0800
Subject: [PATCH 1262/1374] media: intel-ipu4: [VIRT] Increase
 MAX_ISYS_VIRT_STREAM to 35

In UOS, the max number of IPU stream device that been exposed is 34
and SOS created 35 stream devices. By incresing MAX_ISYS_VIRT_STREAM
to 35, UOS expose all the stream devices that been created by SOS.

This enables UOS to do 8 camera concurrent streaming whereas
previously UOS only able to do 7 camera concurrent streaming.

Change-Id: I99cde4a26612a267730451a1503c41c8ddd1fdab
Signed-off-by: Gopal, Puunithaaraj <puunithaaraj.gopal@intel.com>
---
 drivers/media/pci/intel/virtio/intel-ipu4-virtio-common.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/pci/intel/virtio/intel-ipu4-virtio-common.h b/drivers/media/pci/intel/virtio/intel-ipu4-virtio-common.h
index 3edab42..b5cc8dd 100644
--- a/drivers/media/pci/intel/virtio/intel-ipu4-virtio-common.h
+++ b/drivers/media/pci/intel/virtio/intel-ipu4-virtio-common.h
@@ -18,7 +18,7 @@
 #define MAX_ENTRY_FE 7
 #define MAX_STREAM_DEVICES 64
 #define MAX_PIPELINE_DEVICES 1
-#define MAX_ISYS_VIRT_STREAM 34
+#define MAX_ISYS_VIRT_STREAM 35
 
 #define phys_to_page(x) pfn_to_page((x) >> PAGE_SHIFT)
 
-- 
2.7.4

